lib/phoenix_test/playwright/serialization.ex

```diff
@@ -63,4 +63,27 @@ defmodule PhoenixTest.Playwright.Serialization do
   defp to_atom(nil), do: raise(ArgumentError, message: "Unable to convert nil into an atom")
   defp to_atom(s) when is_binary(s), do: String.to_atom(s)
   defp to_atom(a) when is_atom(a), do: a
+
+  def camel_case_keys(enum) when is_map(enum) or is_list(enum) do
```

Comment:
I'm not sure this is entirely in the spirit of the library, but I find it more natural to write `|> screenshot("foo.png", full_page: true)` rather than `fullPage: true`, even though Playwright documents the option in the camelCase version.

Reply:
I agree with you. But I would rather do this globally for all messages we send to playwright to keep things consistent.

So remove from this PR and move e.g. to `Port`.
Similar to this: https://github.com/mechanical-orchard/playwright-elixir/blob/4d0f740e972066310fa1ca133e3ae6ffab06b1ef/lib/playwright/sdk/channel/message.ex#L36

Reply:
#11 should do that if you would care to have a look.

Reply:
Done